src/main/java/core/basesyntax/FigureToDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureToDraw {
```

Comment:
Using 'Figure' in this name is against Abstraction OOP principle
```suggestion
public interface Drawable {
```